pkg/compiler/codegen.go

```diff
@@ -583,12 +584,16 @@ func (c *codegen) Visit(node ast.Node) ast.Visitor {
 						} else {
 							c.scope.newLocal(id.Name)
 						}
-						c.registerDebugVariable(id.Name, t.Type)
+						if !multiRet {
```

Comment:
There has to be something registered even in this case, otherwise debugging this code won't be fun. See #830 as well.

Reply:
IIUC, current debug info specification still doesn't allow to declare multi-return functions even if they are not exported. https://github.com/neo-project/neo-debugger/issues/17 is not resolved and I've added a comment to check the status, see https://github.com/ngdenterprise/design-notes/issues/4#issuecomment-1216664476.